eth_worker/eth_manager/utils.py

```diff
@@ -6,22 +6,22 @@
 import config
 
 
-def execute_synchronous_celery(signature, queue='celery'):
-    async_result = signature.apply_async(queue=queue)
+def execute_synchronous_celery(signature):
+    result = signature.apply()
```

Comment:
ohhhhhhhh like here?

Reply:
💯

Reply:
Note: I did revert this to the previous. Was getting a bit of funky behaviour and want to limit the scope of this PR